From d0b24f87cad06363baf07fe9f59da6371c7a7c8f Mon Sep 17 00:00:00 2001
From: Bogdan Hamciuc <bogdan.hamciuc@freescale.com>
Date: Mon, 8 Apr 2013 17:31:15 +0300
Subject: [PATCH 0955/1383] dpaa_eth: Add compatible string to parent device

Make it so our probing function is called at boot-time.

Signed-off-by: Bogdan Hamciuc <bogdan.hamciuc@freescale.com>
Change-Id: Iaea6116e679216a6d8432643a55c2036e4e07b1a
Reviewed-on: http://git.am.freescale.net:8181/1264
Reviewed-by: Fleming Andrew-AFLEMING <AFLEMING@freescale.com>
Tested-by: Fleming Andrew-AFLEMING <AFLEMING@freescale.com>
(cherry picked from commit 89ce8a974da4f0ba6a16c4a3ff0e64a3da8709b0)
Signed-off-by: Scott Wood <scott.wood@nxp.com>
[Original patch taken from QorIQ-SDK-V2.0-20160527-yocto]
Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
---
 arch/powerpc/platforms/85xx/corenet_generic.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/powerpc/platforms/85xx/corenet_generic.c b/arch/powerpc/platforms/85xx/corenet_generic.c
index 1ecaf83..92be914 100644
--- a/arch/powerpc/platforms/85xx/corenet_generic.c
+++ b/arch/powerpc/platforms/85xx/corenet_generic.c
@@ -94,6 +94,9 @@ static const struct of_device_id of_device_ids[] = {
 		.compatible	= "simple-bus"
 	},
 	{
+		.compatible	= "fsl,dpaa",
+	},
+	{
 		.compatible	= "mdio-mux-gpio"
 	},
 	{
-- 
2.8.1

